joomla/jissues
# Debugging and Logging improvements

### Summary of Changes

* Move HTML stuff from the debugger to a dedicated class (still looks like hell...)
* Parse the log files generated by Monolog (all but the PHP log), and display them in a more eye friendly manner.

### Testing Instructions

* Check that debugging still works and everything shows as before.
* Check that log file display has been improved - suggestions welcome.

**Note** I tested with my local files as well as the files available to me on the server. I noted that the server files are not quite up to date and some are really huge (e,g, the `cron.log` counts > 50mb)...
 
### Screens

**Before**

![joomla issue tracker 2016-08-04 09-04-47](https://cloud.githubusercontent.com/assets/33978/17404796/9e39fba4-5a22-11e6-9b58-2daffb1a3c7e.png)

**After**

![joomla issue tracker 2016-08-03 19-02-03](https://cloud.githubusercontent.com/assets/33978/17403581/12a27278-5a1e-11e6-8b96-1baefa681785.png)
![joomla issue tracker 2016-08-03 19-02-11](https://cloud.githubusercontent.com/assets/33978/17403579/12912022-5a1e-11e6-8e5e-dcb7e307f621.png)

![joomla issue tracker 2016-08-03 19-00-42](https://cloud.githubusercontent.com/assets/33978/17403578/1291d670-5a1e-11e6-877e-eb7e6549d8e2.png)
![joomla issue tracker 2016-08-03 18-49-29](https://cloud.githubusercontent.com/assets/33978/17403583/12b87c6c-5a1e-11e6-88f1-e178a2ecb022.png)

![joomla issue tracker 2016-08-03 19-01-49](https://cloud.githubusercontent.com/assets/33978/17403582/12a29096-5a1e-11e6-8bbc-0a8899afc9c2.png)
![joomla issue tracker 2016-08-03 19-15-43](https://cloud.githubusercontent.com/assets/33978/17403580/129a160a-5a1e-11e6-8f38-1e6cf1224426.png)
